Tests/Imagine/Filter/Loader/UpscaleFilterLoaderTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Liip\ImagineBundle\Tests\Filter;
+
+use Imagine\Gd\Image;
+use Imagine\Gd\Imagine;
+use Liip\ImagineBundle\Imagine\Filter\Loader\UpscaleFilterLoader;
+use Liip\ImagineBundle\Tests\AbstractTest;
+
+/**
+ * @covers Liip\ImagineBundle\Imagine\Filter\Loader\UpscaleFilterLoader
+ */
+class UpscaleFilterLoaderTest extends AbstractTest
+{
+    public function testLoad()
+    {
+        $loader = new UpscaleFilterLoader();
+        $imagine = new Imagine();
+        $image = $imagine->open(__DIR__.'/../../../Fixtures/assets/square.png');
```

Comment:
could you name an image so that it contains width and height square-200x200.png. and rename the test so it is clear what it test.